src/components/progress-flow/progress-flow-item/progress-flow-item.scss

```diff
@@ -133,7 +133,7 @@
     z-index: z-index.$limel-progress-flow-divider;
```

Comment:
Regarding the failing tests: it seems the tests that depend on Org or Repo secrets are failing, because your code is coming from a fork. Which makes sense, since code from a fork could probably get hold of those secrets if they were supplied to the workflow; it definitely could if the workflow files used are those from the fork…

Since this is a trivial fix, I'll just run the tests locally to make sure they pass, but I'll have to look into if there's some better way at some point 🙂

Reply:
Build, tests, and linting all pass 👍